### ISSUE TITLE
botpress-platform-webchat with HITL id error

### ISSUE THREAD

**Issue by obirije** [botpress/botpress-platform-webchat#5]
botpress-platform-webchat returns an id error when trying to chat from the human in the loop (HITL) module web UI 

rror: Unhandled Rejection in Promise:  Promise {
  _bitField: 18087936,
  _fulfillmentHandler0:
   TypeError: Cannot read property 'id' of undefined
       at _callee$ (C:\Users\user\botpress\node_modules\botpress-platform-webchat\bin\webpack:\src\socket.js:36:38)

**Comment by monarch0111** [comment]
I have tried to fix this issue, although it is not the best way.
I'm still into figuring out the code thoroughly. 

If you want you can use this for now. I'll update once the code is ready to be merged into the original repo: [Dirty fix for issue 5](https://github.com/monarch0111/botpress-platform-webchat/commit/d004a9691b8f1ff75546738cd8e2b478adbe0dd7)

**Comment by monarch0111** [comment]
Hi @slvnperron,
I tried to fork and make changes to resolve the issue, but i'm not able to locally install the changed `botpress-platform-webchat` plugin. 

I tried with below steps and commands.
1. Remove current `botpress-platform-webchat`
```bash
botpress uninstall botpress-platform-webchat
```
2. Install local `botpress-platform-webchat`
```bash
botpress-platform-webchat> npm run compile
myBot> botpress install ~/botpress/botpress-platform-webchat #path-to-updated-module
```
3. Start myBot
```bash
botpress start
```
Output
```bash
16:26:30 - info: Starting botpress version 1.1.2
16:26:31 - info: Loaded botpress-analytics, version 2.2.4
16:26:31 - info: Loaded botpress-api.ai, version 2.1.5
16:26:31 - info: Loaded botpress-hitl, version 1.0.10
16:26:31 - verbose: [UMM] Enabled for facebook
16:26:31 - info: Loaded botpress-messenger, version 2.2.9
(node:80840) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Cannot find module 'botpress'
16:26:36 - debug: botpress-analytics: recompiling analytics
```

After uninstalling the local modified module and installing the current release module via 
```bash
botpress install botpress-platform-webchat
```
it works fine but the issue #5 is persistent.

Please help.

**Comment by monarch0111** [comment]
Hi @slvnperron,
Can you please look into this? Your help would be really appreciated. 
I'm not able to debug it further as error message doesn't list down specific details about the issue.
